### ISSUE TITLE
Prepare release notes for OFN v2.0.0 and create user guide issue with details

### ISSUE THREAD

**Issue by luisramos0** [openfoodfoundation/openfoodnetwork#3598]
## What is the problem we are solving 
We need to compile v2 release notes and with that info create an issue to update the different parts of the user guide that change in v2.

## Success factors = expected outcome 
Release notes are ready to be published and an issue exists listing all the parts where the user guides needs to be updated.

### Draft list of topics to be covered
* on_demand
* shipping categories in products and shipping methods
* default stock location
* etc

**Comment by luisramos0** [comment]
@sauloperez fyi, this was one of the actions in todays meeting

**Comment by mkllnk** [comment]
This will be work in progress for a while. But I made a start:
https://github.com/openfoodfoundation/openfoodnetwork/releases/tag/untagged-3cef283e54425c8541b5
:tada: 

@luisramos0 Can you add a paragraph around shipping categories and shipping methods? I'm not 100% sure what is affected.

Regarding the default stock location: Is it visible to people? Do they need to know about it?

**Comment by mkllnk** [comment]
@RachL If you come across an important change in the Spree upgrade then this issue is the place to mention it. We will collect all changes here to include them in the release notes.

Would you like to track a task list for documentation updates here as well? Or would you like a separate issue for that?

**Comment by luisramos0** [comment]
ahaha, we have a name for it 🎉 
I'll add some things I remember :+1:

**Comment by luisramos0** [comment]
ok, I have added a few points: shipping categories, shipping zones,stock locations and stock items (that is really why this is a major release), checkout workflow and adjustments. Is there anything else?

re stock logic: when a variant is on_demand, the stock levels are not seen anywhere (we removed the stock management page here #3613), only when the user unchecks the on_demand box, the stock is shown. Maybe we need to clarify that in the text.

**Comment by mkllnk** [comment]
Ah, good one. I updated the draft. I'm wondering if we should talk that much about invisible changes that may be used in the future. I think we need one list of important changes that people need to know. We can also have a list of behind-the-scenes changes to give people a perspective what may happen in the future. But mentioning that is dangerous. There will be people demanding these new features while others just don't want change.

**Comment by luisramos0** [comment]
yes, that seems reasonable. we can move this to a wiki or something. I just wanted to put it out there.
the problem is that we will have to say that the changes to shipping categories and zones are useless right now :-(

**Comment by mkllnk** [comment]
#### Changes to be documented in the user guide

* Shipping category is now mandatory

**Comment by mkllnk** [comment]
Yes, we can delete them. It's all in Git if we need to recover information. If there is a transition phase in which people may need both information, we can have sections for v1 and v2.

**Comment by luisramos0** [comment]
ok, I cleaned up most tech docs into text that makes sense post spree upgrade.

Now we have a few pages about the upgrade process that are just history, I vote deleting them to keep the wiki simple and small but I need your consent @sauloperez 
Pages to be deleted:
https://github.com/openfoodfoundation/openfoodnetwork/wiki/Spree-2.0-upgrade
https://github.com/openfoodfoundation/openfoodnetwork/wiki/Team-catch-ups--%F0%9F%92%AC
https://github.com/openfoodfoundation/openfoodnetwork/wiki/Upgrade-catch-up-May-9%2C-2018
https://github.com/openfoodfoundation/openfoodnetwork/wiki/Upgrade-catch-up-Jul-19%2C-2018
https://github.com/openfoodfoundation/openfoodnetwork/wiki/Spree-Upgrade:-Development-%E2%9B%91
https://github.com/openfoodfoundation/openfoodnetwork/wiki/Spree-Upgrade:-Getting-to-version-2.0.4-%E2%9B%B0
https://github.com/openfoodfoundation/openfoodnetwork/wiki/Spree-Upgrade:-Notes-about-Maikel's-suggested-approach/59a7f6c4aa44dff8f00a1a90fb821ed3e9416fc9

**Comment by luisramos0** [comment]
regarding the release notes:
- Shipping Categories - I verified we are not filtering shipping methods on checkout and shipping categories do not have to match product categories for the checkout to work. I left the entry in the release notes to announce the new fields but simplified the explanation: "Shipping Categories have no influence on the checkout process."
- Zones in Shipping Methods - I have confirmed the list of countries in the checkout page is the list of countries of the checkoutzone of the instance. So, if you make the zone of a shipping method be some other zone that does not contain the country the user selects, checkout will fail. I make a quick note about this on the release notes to clarify (and so that we don't forget :-))
- I have added #1631 to the release notes

Feel free to further reduce or change yourself!

I think it will take longer to update the user guide. Shall we close this issue now that release notes are ready to go?
